protocol-definitions/MC.yaml

```diff
@@ -222,3 +222,96 @@ methods:
             8 - FREE_NATIVE_MEMORY_SIZE
             9 - FREE_NATIVE_MEMORY_PERCENTAGE
     response: {}
+  - id: 5
+    name: getMemberConfig
+    since: 2.0
+    doc: |
+      Gets the effective config of a member rendered as XML.
```

Comment:
What about YAML?

Reply:
We only show the effective configuration. AFAIK it is only available as XML.